### PR TITLE
Provide direct download links to all Android APK architectures

### PR DESCRIPTION
**Pull request by rollerozxa**
Rather than just providing a link to the ARM 32-bit APK like that is all that's available.

### CODE REVIEW

**Comment by rollerozxa** [comment]
https://github.com/minetest/minetest/issues/9107, I assume? Unsure if it still occurs, haven't experienced it myself.

Google Play and F-Droid already distributes arm64 binaries to all 64-bit phones, and the 64-bit version obviously performs better too (assumedly also uses GC64). So I'm unsure if there's much purpose to hiding direct downloads for it.

**Comment by sfan5** [comment]
> [minetest/minetest#9107](https://github.com/minetest/minetest/issues/9107), I assume?

yes


> the 64-bit version obviously performs better too (assumedly also uses GC64).

in theory yes
in practice if the loading screen is stuck making barely any progress for a minute you lose 30% of players (made up figure). it doesn't matter how great the game performs to players who don't make it there.

**Comment by sfan5** [comment]
Please rebase, this issue has been adressed as of https://github.com/minetest/minetest/pull/14288

**Comment by rollerozxa** [comment]
> Please rebase, this issue has been adressed as of [minetest/minetest#14288](https://github.com/minetest/minetest/pull/14288)

I hope so considering a direct download link to the ARM64 apk was already linked on the download page in the time after I had opened this PR. x) But this PR also adds download links to the x86 and x86_64 apks, so this PR is still relevant.

File: downloads.html [hunk 1]
```diff
@@ -80,13 +80,13 @@ <h2>Android</h2>
         <ul>
           <li>
             ARM:
-            <a href="https://github.com/minetest/minetest/releases/download/5.7.0/app-arm64-v8a-release.apk">64-bit</a> -
-            <a href="https://github.com/minetest/minetest/releases/download/5.7.0/app-armeabi-v7a-release.apk">32-bit</a>
+            <a href="https://github.com/minetest/minetest/releases/download/5.9.1/minetest-5.9.1-arm64-v8a-release.apk">64-bit</a> -
```
Comment (appgurueu) [inline downloads.html:83]: Dead link? Looks like it should be https://github.com/minetest/minetest/releases/download/5.9.1/minetest-5.9.1-arm64-v8a.apk (minus the `release` part).

**Pull request merged**